src/test/java/org/alfresco/rest/api/tests/TestPeople.java

```diff
@@ -2296,6 +2295,42 @@ public void removeAvatar() throws IOException, PublicApiException{
         
     }
 
+    @Test
+    public void testListPeopleWithCapabilities() throws Exception

```

Comment:
Please add a test or update the existing test for getPeopleById to see that we have the capabilities also there.

Reply:
Thanks for the observation. Added a validation in the existing test in this [commit ](https://github.com/Alfresco/alfresco-remote-api/pull/47/commits/bc3910ce998ca728e4a82d53a1eebe50be8b58b2).